hphp/system/php.txt

```diff
@@ -52,7 +52,7 @@ hphp/system/php/spl/file_handling/SplFileObject.php
 hphp/system/php/spl/file_handling/SplTempFileObject.php
 
 hphp/system/php/lang/ArrayAccess.php
-hphp/system/php/lang/Serializeable.php
+hphp/system/php/lang/Serializable.php
```

Comment:
I'm pretty sure this change will be dropped (it's a single commit, not a big deal)